script.js

```diff
@@ -1,4 +1,108 @@
+var GAME_STATE_DICE_ROLL = "GAME_STATE_DICE_ROLL";
+var GAME_STATE_CHOOSE_DICE_ORDER = "GAME_STATE_CHOOSE_DICE_ORDER";
+var GAME_STATE_CHOOSE_COMPARE_SCORES = "GAME_STATE_COMPARE_SCORES";
+var gameState = GAME_STATE_DICE_ROLL;
+
+var currentPlayerRolls = [];
+
+var currentPlayer = 1;
+var allPlayersScore = [];
+
+var playerWins = [0, 0, 0];
+
+var rollDice = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal) + 1;
+  return randomInteger;
+};
+
+var rollDiceForPlayer = function () {
+  var counter = 0;
+  while (counter < 2) {
+    currentPlayerRolls.push(rollDice());
+    counter = counter + 1;
+  }
+  return `Welcome, Player ${currentPlayer}. <br><br>You rolled:<br>Dice 1: ${currentPlayerRolls[0]}| Dice 2: ${currentPlayerRolls[1]}.<br><br>Now, please input '1' or '2'`;
+};
+
+var getPlayerScore = function (playerInput) {
+  var playerScore;
+  if (playerInput == 1) {
+    playerScore = Number(
+      String(currentPlayerRolls[0]) + String(currentPlayerRolls[1])
+    );
+    allPlayersScore.push(playerScore);
+    currentPlayerRolls = [];
+    return `Player ${currentPlayer}, your chosen value is : ${playerScore}`;
+  }
+  if (playerInput == 2) {
+    playerScore = Number(
+      String(currentPlayerRolls[1]) + String(currentPlayerRolls[0])
+    );
+    allPlayersScore.push(playerScore);
+    currentPlayerRolls = [];
+    return `Player ${currentPlayer}, your chosen value is : ${playerScore}`;
+  }
+};
+
+var comparePlayersScores = function () {
+  var compareMessage = `Player 1 score: ${allPlayersScore[0]}<br>Player 2 score: ${allPlayersScore[1]}`;
+  if (allPlayersScore[0] > allPlayersScore[1]) {
```

Comment:
Might be a good idea to initiate 
```
var player1Score = allPlayersScore[0];
var player2Score = all PlayersScore[1];

if (player1Score > player2Score) { // logic you currently have}
```
Reason being, it's not very intuitive to the reader that `allPlayersScore[0/1]` refers to which player's score